### PR TITLE
Revise primary tests (minor)

### PR DESCRIPTION
**Pull request by awwad**
Mostly cleanup and addition of some defensive code.

Commit a95862c re-orders some imports to put them in an order I think is probably best for imports going forward: least specific and least project related to most specific and most project related.

### CODE REVIEW

**Comment by awwad** [comment]
Rebased. This can be reviewed now.

File: tests/test_primary.py [hunk 1]
```diff
@@ -14,22 +14,23 @@
...
-import tuf.keys # to validate a signature
-
 import unittest
 import os.path
 import time
 import copy
 import shutil
 import hashlib
+
+from six.moves.urllib.error import URLError
```
Comment (vladimir-v-diaz) [inline tests/test_primary.py:24]: Was this an organizational change?  Did you want to organize the stdlib modules first, then libraries, etc.?

Comment (awwad) [reply tests/test_primary.py:24]: Yeah. I've landed on most general and least project related to most specific and most project related in terms of import order. Thoughts on that?

**Comment by awwad** [comment]
Am thinking of removing the globals in test_primary by initializing the values in module-level code rather than inside functions.

That would slow importing this module down significantly, but realistically, things aren't going to import test_primary.

File: tests/test_primary.py (hunk 1 above)
Comment (vladimir-v-diaz) [reply tests/test_primary.py:24]: I agree with your organization of module imports.  It should probably be explained in the PR summary.

**Comment by awwad** [comment]
This:
```python
# Load the private key for this Primary ECU.
primary_ecu_key = uptane.common.canonical_key_from_pub_and_pri(
    demo.import_public_key('primary'), demo.import_private_key('primary'))

# Generate a trusted initial time for the Primary.
clock = tuf.formats.unix_timestamp_to_datetime(int(time.time()))
clock = clock.isoformat() + 'Z'
tuf.formats.ISO8601_DATETIME_SCHEMA.check_match(clock)

# Load the public timeserver key.
key_timeserver_pub = demo.import_public_key('timeserver')
key_timeserver_pri = demo.import_private_key('timeserver')
```

This means that setUpModule will be empty:
```python
def setUpModule():
  """
  This is run once for the full module, before all tests.
  """
  destroy_temp_dir()
```

**Comment by vladimir-v-diaz** [comment]
Okay, I don't see anything wrong with the cleanup PR.  Safe to merge/

**Pull request merged**